### ISSUE TITLE
Implement an Aria-Live Alert Sub-System

### ISSUE THREAD

**Issue by myvuuu** [Workday/canvas-kit#1582]
## 🚀 Feature Proposal

Create an aria-live alert sub-system for Canvas components to pass messages and alerts to that will be announced by a screen reader. This aria-live sub-system should be implemented in:

- [ ] Toast
- [ ] Breadcrumbs
- [ ] Pagination

## Motivation

This will help us enhance the accessibility features for Canvas Toast, which currently does not announce alerts to screen readers.

**Comment by jaclynjessup** [comment]
These components: Toast, Breadcrumbs, and Pagination - should be able to work with an aria-live sub-system even if built by another team.

Based on the closed Spike #1630 it was deemed we would not build this but now we are. This will be put on hold till @NicholasBoll is back & further discussion will be held.

**Comment by hailthekid** [comment]
for documentation purposes, this thread discusses woes of toast component 
https://workday-dev.slack.com/archives/C8X0W5AQH/p1720451659810149?thread_ts=1720270800.816499&cid=C8X0W5AQH
